types/office-js-preview/index.d.ts

```diff
@@ -13298,27 +13300,33 @@ declare namespace Office {
      */
     interface Diagnostics {
         /**
-         * Gets a string that represents the name of the Office application.
-         *
-         * A string that can be one of the following values: `Outlook`, `OutlookWebApp`, `OutlookIOS`, or `OutlookAndroid`.
+         * Gets a string that represents the name of the Outlook client.
```

Comment:
Just a suggestion. Ignore if you want.

```suggestion
         * Gets a string that represents the type of Outlook client.
```